tsconfig.base.json

```diff
@@ -7,6 +7,7 @@
     "moduleResolution": "node",
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
+    "strictPropertyInitialization": false,
```

Comment:
No! Do NOT disable checks simply because you don't know how to fix them. Ask for help!!!